rt/transports/http-jetty/src/main/java/org/apache/cxf/transport/http_jetty/JettyHTTPServerEngine.java

```diff
@@ -456,10 +459,23 @@ public synchronized void addServant(URL url, JettyHTTPHandler handler) {
                             //then it need be on top of JettyHTTPHandler
                             //set JettyHTTPHandler as inner handler if
                             //inner handler is null
-                            ((SecurityHandler)h).setHandler(handler);
+                            //((SecurityHandler)h).setHandler(handler);
```

Comment:
It is commented out accidentally or not needed?

Reply:
this isn't needed, a new commit removed it